tests/test_repository_lib.py

```diff
@@ -86,24 +81,25 @@ def tearDownClass(cls):
     tuf.roledb.clear_roledb(clear_all=True)
     tuf.keydb.clear_keydb(clear_all=True)
 
-    shutil.rmtree(cls.temporary_directory)
-
 
   def setUp(self):
+    # We are inheriting from custom class.
+    unittest_toolbox.Modified_TestCase.setUp(self)
```

Comment:
This pattern looks odd, isn't the normal way to do this with `super().setUp(self)` ?